<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>711</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane{
min-width:70px;
min-height:25px;
border-top: 2px solid;

}

QTabBar::tab {

min-width:70px;

min-height:25px;

color: white;

font:15px &quot;Microsoft YaHei&quot;;

border: 0px solid;



}

QTabBar::tab:selected{

min-width:70px;

min-height:25px;
color: #4796f0;

font:15px &quot;Microsoft YaHei&quot;;

border: 0px solid;

border-bottom: 2px solid;

border-color: #4796f0;

}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font:15px &quot;Microsoft YaHei&quot;;</string>
      </property>
      <property name="text">
       <string>Number of Patients</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(96,104,132);
font-family: 微软雅黑;
color:white
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font:15px &quot;Microsoft YaHei&quot;;</string>
      </property>
      <property name="text">
       <string>Number of Doctors</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(96,104,132);
font-family: 微软雅黑;
color:white
</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>551</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Output</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>471</width>
        <height>291</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(255, 255, 255, 60);
	color:white;
	border-radius: 8px;
	font-family: 微软雅黑;
	text-align: center;
}
QPushButton:hover{                    
	background:rgb(255, 255, 255, 100);
}
QPushButton:pressed{
	background: darkgray;
}</string>
      </property>
      <property name="text">
       <string>Output Chart</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>最小化</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background:#6C6C6C;
    color:white;
    box-shadow: 1px 1px 3px rgba(0,0,0,0.3);font-size:16px;border-radius: 8px;font-family: 微软雅黑;
}
QPushButton:hover{
    background:#9D9D9D;
}
QPushButton:pressed{
    border: 1px solid #3C3C3C!important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>最大化及还原</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background:green;
    color:white;
    box-shadow: 1px 1px 3px rgba(0,0,0,0.3);font-size:16px;border-radius: 8px;font-family: 微软雅黑;
}
QPushButton:hover{
    background:#7bcd68;
}
QPushButton:pressed{
    border: 1px solid #3C3C3C!important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>关闭程序</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background:#CE0000;
    color:white;
    box-shadow: 1px 1px 3px rgba(0,0,0,0.3);font-size:16px;border-radius: 8px;font-family: 微软雅黑;
}
QPushButton:hover{
    background:#FF2D2D;
}
QPushButton:pressed{
    border: 1px solid #3C3C3C!important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>重启程序</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background:#abac30;
    color:white;
    box-shadow: 1px 1px 3px rgba(0,0,0,0.3);font-size:16px;border-radius: 8px;font-family: 微软雅黑;
}
QPushButton:hover{
    background:yellow;
}
QPushButton:pressed{
    border: 1px solid #3C3C3C!important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
